### PR TITLE
codelimit 0.6.0 (new formula)

### PR DESCRIPTION
**Pull request by robvanderleek**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/c/codelimit.rb
Comment (p-linnane) [inline Formula/c/codelimit.rb:8]:
```suggestion
license "ISC"

```

File: Formula/c/codelimit.rb
Comment (p-linnane) [inline Formula/c/codelimit.rb:14]:
```suggestion

```

Comment (p-linnane) [reply Formula/c/codelimit.rb:14]: The bottle block will be generated by our workflows if this PR is accepted.

File: Formula/c/codelimit.rb
Comment (p-linnane) [inline Formula/c/codelimit.rb:41]:
```suggestion

```

File: Formula/c/codelimit.rb
Comment (p-linnane) [inline Formula/c/codelimit.rb:116]:
```suggestion

```

File: Formula/c/codelimit.rb
Comment (p-linnane) [inline Formula/c/codelimit.rb:131]:
```suggestion

```

File: Formula/c/codelimit.rb
Comment (p-linnane) [inline Formula/c/codelimit.rb:156]:
```suggestion

```

File: Formula/c/codelimit.rb
Comment (p-linnane) [inline Formula/c/codelimit.rb:173]:
```suggestion

```

File: Formula/c/codelimit.rb
Comment (p-linnane) [inline Formula/c/codelimit.rb:15]:
```suggestion
  depends_on "python-certifi"
  depends_on "python-typing-extensions"
  depends_on "pygments"
  depends_on "python@3.11"
  depends_on "six"
```

File: Formula/c/codelimit.rb
Comment (p-linnane) [inline Formula/c/codelimit.rb:1]: We'll need to add this to `pypi_formula_mappings.json` as well for the excluded resources.

**Review by p-linnane — CHANGES_REQUESTED** [review]

File: Formula/c/codelimit.rb
Comment (robvanderleek) [reply Formula/c/codelimit.rb]: I've added these dependencies.

File: Formula/c/codelimit.rb
Comment (robvanderleek) [reply Formula/c/codelimit.rb:1]: I've added the excluded resources.

Comment (p-linnane) [reply Formula/c/codelimit.rb:1]: Thanks. Can you please squash all the commits into one with the message `codelimit 0.6.0 (new formula)`?

**Comment by p-linnane** [comment]
Thanks for the PR. Running `brew style --fix codelimit` once more should fix the last style error. Once that's done, can you please squash all the commits into one with the message `codelimit 0.6.0 (new formula)`?

**Comment by robvanderleek** [comment]
> Thanks for the PR. Running `brew style --fix codelimit` once more should fix the last style error. Once that's done, can you please squash all the commits into one with the message `codelimit 0.6.0 (new formula)`?

I've squashed all commits 🙂  Thanks for the support.

**Comment by robvanderleek** [comment]
Hi @p-linnane 

Is there any open review issue holding this PR back? Or will it be merged eventually?

**Review by p-linnane — APPROVED** [review]

**Comment by p-linnane** [comment]
My mistake, I thought this finished running. Let me fix that.

**Comment by p-linnane** [comment]
Also for future reference, branches from organizations cause issues with our merge process. Please open them from normal forks instead.

**Comment by p-linnane** [comment]
@carlocab Can you take a look here? Not sure why the replacement workflow says this isn't mergable.

**Comment by robvanderleek** [comment]
It seems I can't enable "Allow edits from maintainers" for this PR anymore. It was created from a [GH org fork](https://github.com/getcodelimit/homebrew-core). Let me know if I can help, no problem to open a new PR from a user fork if needed.

Thanks for all the support.

**Comment by p-linnane** [comment]
If you can open a new PR from a user fork we can probably get this merged today.

**Comment by robvanderleek** [comment]
Hi @p-linnane 

I've opened a PR from my own (user) fork: https://github.com/Homebrew/homebrew-core/pull/140506

KR, Rob

**Pull request closed without merging**
